client/src/components/NavigationBar.tsx

```diff
@@ -3,17 +3,23 @@ import Nav from "react-bootstrap/Nav";
 import Navbar from "react-bootstrap/Navbar";
 import Button from "react-bootstrap/Button";
 import { Home, ClipboardCheck, ClipboardList, User2 } from 'lucide-react';
+import { useAuth } from "../AuthContext";
 
 export default function NavigationBar() {
   const navigate = useNavigate();
-  const is_loggedin = true; //TODO: login treatment
+  const { user, logout } = useAuth();
```

Comment:
Nitpicky but it's slightly unclear what `user` is here, could rename this to `userId` or `userObject` depending on which it is

Reply:
'user' is an object property set when the login() function is called. It should contain email:string, id:string, and name:string properties (i.e. user.name can be called in other routes this way!), but it seems that I am missing the email:string property from the interface 'User'. I'll update that in a commit before merging